aiidalab_launch/profile.py

```diff
@@ -56,6 +56,7 @@ class Profile:
     system_user: str = "aiida"
     image: str = _DEFAULT_IMAGE
     home_mount: str | None = None
+    custom_mounts: list[str] = field(default_factory=lambda: [])
```

Comment:
I am new to this advanced `dataclass` business, but supposedly I cannot simply initialize to empty list here?

Reply:
Thanks, that's what I suspected, so essentially it's the same gotcha as using an empty list as function optional parameter.

https://docs.python.org/3/library/dataclasses.html#mutable-default-values